src/sentry/integrations/vsts/notify_action.py

```diff
@@ -46,13 +103,23 @@ def generate_footer(self, rule_url):
     @transaction_start("AzureDevopsCreateTicketAction.after")
     def after(self, event, state):
         organization = self.project.organization
-        integration = self.get_integration()
+        try:
+            integration = self.get_integration()
+        except Integration.DoesNotExist:
+            # Integration removed, rule still active.
+            return
+
         installation = integration.get_installation(organization.id)
+
+        self.data["title"] = event.title
         self.data["description"] = self.build_description(event, installation)
 
         def create_issue(event, futures):
```

Comment:
@ceorourke seems like this function is almost the same as Jira. Wondering if we can refactor this into the `TicketEventAction`.